client/css/style.css

```diff
@@ -678,9 +678,15 @@ button, .user {
 }
 #chat .part .from:before,
 #chat .quit .from:before {
-    font-family: FontAwesome;
+	font-family: FontAwesome;
 	content: "\f08b";
 	color: #FF4136;
+	display: inline-block;
```

Comment:
Inline elements can't be transformed, and pseudo selectors are automatically inline.

http://stackoverflow.com/questions/9779919/css-rotate-a-pseudo-after-or-before-content